doc/design-balance-lore/VARIANTS.md

```diff
@@ -0,0 +1,41 @@
+# Variants
+
+## What are variants?
+
+Variants are CDDAs solution to allowing the same item have different names. It's used in guns to have generic or specific names changeable with a setting, or in items such as clothing to have lots of different colors without having a bunch of essentially duplicate entries for an item. They're a great way to add in extra variety to the game at a low cost to both effort, and file sizes. They were added in [51613](https://github.com/CleverRaven/Cataclysm-DDA/pull/51613) and [46814](https://github.com/CleverRaven/Cataclysm-DDA/pull/46814).
+
+## What aren't variants?
+
+Variants cannot change any stat of an item, besides the name, description, color, sprite, and symbol. They're nothing more than an aesthetic change and has no impact on gameplay. 
+
+# Inclusion Criteria 
+
+## Issues
+
+As cool as variants are, sometimes issues can arise if there's too much variants of an item with nothing to reign them in. As an example, condoms had a few joke flavors that didn't make sense, and snowballed into other items having variants that were nothing more than jokes, such as chewing gum having fart, cat hairball, and zombie flavors. This doc aims to provide contributors with a good idea of what is, and isn't acceptable as variants. 
+
+## Criteria 
+
+### Goals
+
+What your goals for variants are should be adding in variety, not content for the sake of it. Being overly specific in your variants devalues the other variants, while not actually adding any more variety. Variants should be stuff you would reasonably find in the real world. As an example, cigarattes such as Parliments or Newports are commonplace, while Golden Bat cigarettes would be too rare. Don't be overzealous with joke variants, one joke variant alone is not likely to be accepted while multiple surely aren't. There isn't any hard number on the amount of variants that are acceptable, as it can vary greatly from item to item. While this entry is good for a general grasp of what is acceptable, below are some more specific examples of variants you may want to add.
+
+### Foods
+
+Variants for foods generally cover just flavors, as those tend to have little to no calorie difference. You should try to cover a base of flavors without being too specific. Strawberry, banana, raspberry flavours are acceptable as chewing gum flavors, while Crazy Carl's Supreme Strawberry Surprise flavor would not. You also want to avoid overlap, strawberry banana raspberry twist chewing gun doesn't add more variety than the individual variants already do. 
+
+### Clothing
+
+Clothing can benefit greatly from variants, as there's a lot of different clothes that have different colors. However, it shouldn't be too in the weeds for details. Aim for covering basic colors and patterns, while avoiding overly specific ones.  A pink hoodie is fine, but a shrimp pink hoodie or a cream and blue Rorschach polka dot hoodie is not.
```

Comment:
The only reason to use variant for colors over dynamically expanded snippet is that they provide a way to specify different textures; if it is not applicable for item to have different textures (and no one gonna make 30 different textures for gum, i suspect), then it is better to use such snippet

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON/JSON_INFO.md#snippets